Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+bottle = ">=0.12,<0.13"
+jsonschema = ">=2.6,<3"
```

Comment:
Is the SemVer like `~=` operator supported? I'd prefer it over the harder to read `>=,<` operator.

Reply:
IMO the current syntax is much less ambiguous than `~`, it says exactly which range is acceptable. With `~` it's more complicated...

Reply:
Hmm? Whats more complicated about `jsonschema = "~=2.6"`?